unlock-app/src/components/interface/VerificationStatus.tsx

```diff
@@ -65,13 +74,18 @@ export const VerificationStatus = ({ data, sig, hexData }: Props) => {
       <OwnsKey
         loading={queryResults.loading}
         error={queryResults.error}
-        data={queryResults.data}
-        lockAddress={lockAddress}
+        matchingKey={matchingKey}
       />
 
       <p>
         Signed {durationsAsTextFromSeconds(secondsElapsedFromSignature)} ago.
       </p>
+      {matchingKey && identityIsValid && (
```

Comment:
It might be worth while to extract this into its own method `matchingKey && identityIsValid`. It should make this a bit easier to test without needing to render the component